TodoSPA/App/Scripts/todoListCtrl.js

```diff
@@ -31,6 +31,11 @@ angular.module('todoApp')
             $scope.todoList = results;
             $scope.loadingMessage = "";
         }).error(function (err) {
+            if (err.access_token)
+            {
+                // Re-request the token with additional claims
+                adalService.acquireTokenRedirect('/api/TodoList', null, JSON.stringify(err));
```

Comment:
I think it's a good idea to check for the specific error your service in generated rather than making this happen whenever the `access_token` field is populated.